srcpkgs/gamescope/template

```diff
@@ -0,0 +1,55 @@
+# Template file for 'gamescope'
+pkgname=gamescope
+version=3.14.0
+revision=1
+_stb_hash=5736b15f7ea0ffb08dd38af21067c314d6a3aae9
+_vkroots_hash=d5ef31abc7cb5c69aee4bcb67b10dd543c1ff7ac
+_reshade_hash=9fdbea6892f9959fdc18095d035976c574b268b7
+build_style=meson
+configure_args="-Denable_openvr_support=false --force-fallback-for="
+hostmakedepends="glslang pkg-config"
+makedepends="pipewire-devel Vulkan-Headers SPIRV-Headers libX11-devel
+ libXdamage-devel libXcomposite-devel libXrender-devel libXxf86vm-devel
+ libXtst-devel libXres-devel libdrm-devel vulkan-loader wayland-protocols
+ libxkbcommon-devel libcap-devel SDL2-devel hwids wlroots0.17-devel
+ libliftoff-devel glm libXmu-devel libdisplay-info-devel benchmark-devel
+ libavif-devel"
+depends="xorg-server-xwayland"
+short_desc="SteamOS session compositing window manager"
+maintainer="Dexter Gaon-Shatford <dexter.gaonshatford@gmail.com>"
+license="BSD-2-Clause, BSD-3-Clause, MIT"
+homepage="https://github.com/ValveSoftware/gamescope"
+distfiles="https://github.com/ValveSoftware/gamescope/archive/refs/tags/${version}.tar.gz
+ https://github.com/nothings/stb/archive/${_stb_hash}.tar.gz
+ https://github.com/Joshua-Ashton/vkroots/archive/${_vkroots_hash}.tar.gz
+ https://github.com/Joshua-Ashton/reshade/archive/${_reshade_hash}.tar.gz"
+checksum="f7cca4ef1786a34496d436cf233f24d28746da7a0f29d3d3f645ac29f1009381
+ d00921d49b06af62aa6bfb97c1b136bec661dd11dd4eecbcb0da1f6da7cedb4c
+ b4eca5edca75355ea1443ad96fd59b0a407f6a2ce17ef5a8f9849c05fc10155f
+ 165726ad21fbfc221c0363e40b597834068a416a11a1204ae2ac6d13ec161035"
+skip_extraction="${_stb_hash}.tar.gz
+ ${_vkroots_hash}.tar.gz
+ ${_reshade_hash}.tar.gz"
+
+if [ "$CROSS_BUILD" ]; then
+	hostmakedepends+=" wayland-devel"
+fi
+
+post_extract() {
+	vsrcextract -C subprojects/stb ${_stb_hash}.tar.gz
+	cp subprojects/packagefiles/stb/meson.build subprojects/stb
+	vsrcextract -C subprojects/vkroots ${_vkroots_hash}.tar.gz
+	vsrcextract -C src/reshade ${_reshade_hash}.tar.gz
+}
+
+post_patch() {
+	vendor_spirv="../thirdparty/SPIRV-Headers/include/spirv/unified1"
+	system_spirv="${XBPS_CROSS_BASE}/usr/include/spirv/unified1"
+	vsed -i src/meson.build -e "s|${vendor_spirv}|${system_spirv}|"
```

Comment:
```suggestion
	local _vendor_spirv="../thirdparty/SPIRV-Headers/include/spirv/unified1"
	local _system_spirv="${XBPS_CROSS_BASE}/usr/include/spirv/unified1"
	vsed -i src/meson.build -e "s|${_vendor_spirv}|${_system_spirv}|"
```
but this could also be done without variables. it's fine if it's a long line in this case